tests/utils/test_helpers.py

```diff
@@ -579,6 +583,7 @@ def test_enrich_image(case):
     default_images_to_enrich_registry = config.images_to_enrich_registry
     config.images_tag = case.get("images_tag", "0.5.2-unstable-adsf76s")
     config.images_registry = case.get("images_registry", "ghcr.io/")
+    config.vendor_images_registry = case.get("vendor_images_registry", "ghcr.io/")
```

Comment:
Change to a different value than `images_registry` to make sure enriched by the correct config